spec/views/collections/cwic_enabled_opensearch_collections_tagging_spec.rb

```diff
@@ -0,0 +1,106 @@
+require 'spec_helper'
+
+describe 'various provider granule OpenSearch API search behavior'  do
+  include Rack::Test::Methods
+
+  def app
+    Rails.application
+  end
+
+  before (:each) do
+    Flipper.enable(:use_cwic_server)
```

Comment:
so maybe this is the reason you have a library, you can simulate settings for tests?

Reply:
Yes, also a valid reason. Another is that Flipper allows us to be more performant by only checking the value of the ENV var once, and then using the result of that check as long as it's deployed.